draft-ietf-taps-impl.md

```diff
@@ -1208,6 +1038,41 @@ Thanks to Stuart Cheshire, Josh Graessley, David Schinazi, and Eric Kinnear for
 
 --- back
 
+# API Mapping Template {#appendix-mapping-template}
+
+Any protocol mapping for the Transport Services API should follow a common template.
+
```

Comment:
How was the list below selected? What about API primitives that are not in the list, e.g. Rendezvous?

Reply:
The answer is that someone started it like this at some point, very very early in the document's history. But since you ask, this is where things get a bit ugly... we could also ask: "What about the Generic Connection Properties?" and "What about Message Properties?".  Honestly I think that, in principle, ALL of these mappings should be covered here... but it's also a huge amount of work, and it will make the document very long, with questionable benefit (for all the primitives and properties that are indeed contained in minset, the mapping can be back-traced from there). Perhaps we should discuss this at the interim?

Reply:
Yes, my question was general and Rendezvous just an example. Let's discuss this at the interim. 

Depending on how we define the use of the template, what is in covered by the template and what is in specified in a particular mapping may not have to overlap one to one.

Reply:
We can go through the exercise of adding all functions/properties to the template

Reply:
Let's do this, but separately?

Reply:
Agreed.